docs/source/releases.rst

```diff
@@ -1,6 +1,32 @@
 Release History
 ###############
 
+
+v7.4.0 (2023-05-08)
+===================
+
+Device Updates
+--------------
+- `IMS` class:
```

Comment:
Not sure if sphinx will render it differently, but GitHub rich view shows this:
<img width="984" alt="image" src="https://user-images.githubusercontent.com/5139267/236955235-6f4492c7-4095-4929-93b0-b11a65119ac8.png">

Reply:
Sphinx did render it as GitHub did, I'm going to need to adjust it.
I also want to tweak the docstring of `IMS.diff_configuration` to reflect the new return type.